testing/trino-product-tests/src/main/java/io/trino/tests/product/TestGroups.java

```diff
@@ -87,6 +87,7 @@ public final class TestGroups
     public static final String DELTA_LAKE_MINIO = "delta-lake-minio";
     public static final String DELTA_LAKE_GCS = "delta-lake-gcs";
     public static final String DELTA_LAKE_DATABRICKS = "delta-lake-databricks";
+    public static final String DELTA_LAKE_DATABRICKS_HTTP_HMS = "delta-lake-databricks-http-hms";
```

Comment:
Let's give it a more meaningful name: 
`DELTA_LAKE_DATABRICKS_HTTP_HMS` -> `DELTA_LAKE_DATABRICKS_UNITY`